solr_conf/conf/managed-schema

```diff
@@ -104,14 +104,14 @@
     <fieldType name="transcription_text" class="solr.TextField" positionIncrementGap="100" multiValued="true">
     <analyzer type="index">
       <charFilter class="solr.HTMLStripCharFilterFactory"/>
-      <tokenizer class="solr.StandardTokenizerFactory"/>
+      <tokenizer class="solr.WhitespaceTokenizerFactory"/>
```

Comment:
Makes a lot of sense to me that we'd want a less aggressive tokenizer for transcription content, wish I had thought of that when I was working on tuning the search!

Will this cause any problems? e.g., is punctuation at the end of words still removed? (I forget where that typically happens in Solr, and I also don't know how much of a problem it would be for transcription content)

Reply:
Seems like it's actually not, but also doesn't seem to cause any problems. Since this field type is also used for translations, we can make searches like this and see how it behaves differently in that case (description with the standard tokenizer vs. translation with the whitespace—search term `illustrious judge` and `"illustrious judge"` produces this):

![Screenshot 2024-02-22 at 4 54 27 PM](https://github.com/Princeton-CDH/geniza/assets/7234006/f2b4c2de-ccd7-4463-9e6c-30b12763cd69)

It might be worth creating a new field type for translations, actually, but as of now the tokenizer change doesn't seem to be causing problems.